dist/emojionearea.js

```diff
@@ -76,10 +76,17 @@ document = window.document || {};
         if (shortname) {
             friendlyName = shortname.substr(1, shortname.length - 2).replace(/_/g, ' ').replace(/\w\S*/g, function(txt) { return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
         }
+        var fname = '';
+        if (emojioneSupportMode > 4) {
```

Comment:
` if (unicode.uc_base && emojioneSupportMode > 4) {`

Reply:
In some cases the method getTemplate receive unicode var as String, and throws an error when try to access uc_base.

Fixing this point, works correctly

Reply:
@lasdeparq will update :+1: